Sources/Lexer/Token.swift

```diff
@@ -31,13 +31,13 @@ public struct Token: Equatable, SourceEntity, CustomStringConvertible {
     case literal(Literal)
 
     // Keywords
-    case contract, `enum`, `case`, `struct`, `func`, `event`, trait
-    case `init`, fallback
-    case `public`, visible, `mutating`, `external`
-    case `return`, become, `try`, `emit`
+    case `contract`, `enum`, `case`, `struct`, `func`, `event`, `trait`
```

Comment:
Same for these. There's no reason to backtick escape a string unless they are keywords. It's harder to read

Reply:
The commit for this said it is for "consistency". I don't think it's a bad idea, honestly.

Reply:
Okay, that is an option. But that means I will remove backticks from non-keywords, including backticks that were not added by me and they were there from the original code basez

Reply:
uh the only such keyword that I can see there is `emit`, so in terms of 'distance' only backticking keywords is certainly less.